src/frontend/components/app/language-select/language-select.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Button } from '@adminjs/design-system'
+import React, { FC } from 'react'
+import { useSelector } from 'react-redux'
+import { locales } from '../../../../locale'
+import { ReduxState } from '../../../store/store'
+import { useLocalStorage } from '../../../hooks'
+
+const LanguageButtons: FC = () => {
+  const { locale } = useSelector((state: ReduxState) => state)
+  const [loc, setLocale] = useLocalStorage('locale', locale)
+  const { availableLanguages } = locale
+  const handleButton = (lng: string): void => {
```

Comment:
`: void` can be removed since it will be asserted by typescript

Reply:
removed